cmd/hypershift/cmd.go

```diff
@@ -0,0 +1,20 @@
+package hypershift
+
+import (
+	"github.com/spf13/cobra"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+)
+
+// NewCmdAao implements the base aao command
+func NewCmdHypershift(streams genericclioptions.IOStreams) *cobra.Command {
```

Comment:
Is this intended to be `osdctl hypershift` for the command?  

If so, I think this would be better as a noun-verb subcommand for `cluster`, command, given the thing you're going for here.  Maybe something like `osdctl cluster get-cloud-resources` or something along those lines.

If it can be generic-ish, like this, someone else can do something similar for non-hypershift clusters.  I think there's benefits for both.  Obv not needing to gather management cluster info, etc, but getting the resources themselves would be an excellent advantage for both types!

Reply:
Yes I totally agree, I was also thinking that having a similar command (or this command with the right flags) provide similar output for both - given that one of the cluster's that should be collected is the customer cluster anyhow that whole part should be 90% reusable (maybe except the VCP endpoints).